File: max_in_sequence/turingarena-files/Java/skeleton.java
import java.util.Scanner;

abstract class Skeleton {
    private static final Scanner in = new Scanner(System.in);


    abstract int max_index(int n, int a[]);

    public static void main(String[] args) {
        Solution __solution = new Solution();

        // checkpoint
        System.out.printf("%d\n", 0);
        // read n
        int n;
        System.out.flush();
        n = in.nextInt();
        // for i to n {...}
        int[] a;
        a = new int[n];
        for (int i = 0; i < n; i++) {
            // read a[i]
            System.out.flush();
            a[i] = in.nextInt();
        }
        // call index = max_index(n, a)
        int index;
        index = __solution.max_index(n, a);
        // write index
        System.out.printf("%d\n", index);
        // exit
        System.exit(0);
    }
}

File: max_in_sequence/turingarena-files/C/skeleton.c
#include <stdio.h>
#include <stdlib.h>


int max_index(int n, int *a);


int main() {
    // checkpoint
    printf("%d\n", 0);
    // read n
    static int n;
    fflush(stdout);
    scanf("%d", &n);
    // for i to n {...}
    static int *a;
    a = malloc(n * sizeof(*a));
    for (int i = 0; i < n; i++) {
        // read a[i]
        fflush(stdout);
        scanf("%d", &a[i]);
    }
    // call index = max_index(n, a)
    static int index;
    index = max_index(n, a);
    // write index
    printf("%d\n", index);
    // exit
    exit(0);
}
